apps/web-client/src/lib/utils/misc.ts

```diff
@@ -71,3 +75,57 @@ export function matchesName(needle: string, haystack: string) {
 	// Check if all conditions derived from the needle are present in the normalized haystack
 	return needleConditions.every((condition) => normalizedHaystack.includes(condition));
 }
+
+/**
+ * Generates an order file string for Format A (Pearson/PBM).
+ */
+export function generatePearsonFormat(customerId: number, lines: CustomerOrderLineSelection[]): string {
```

Comment:
Case 2. is not an issue. I was confused by the `customerId` reference. I think it should be renamed so that it clearly relates to the "supplier". The `supplier_id` that is passed per CustomerOrderLineSelection seems unneeded and adds to this confusion?